src/applications/appeals/download/containers/Confirmation.jsx

```diff
@@ -57,58 +76,162 @@ const Confirmation = () => {
         <va-alert status="success" uswds>
           <h2 slot="headline">You submitted your Board Appeal request</h2>
           <p>
-            After we’ve completed our review, we’ll mail you a decision packet
-            with the details of our decision.
+            You’ll also receive a copy of your completed form in a confirmation
+            email. After we’ve completed our review, we’ll mail you a decision
+            packet with the details of our decision.
           </p>
         </va-alert>
+
+        {/* <va-summary-box uswds class="vads-u-margin-y--2"> */}
```

Comment:
I couldn't get the `va-summary-box` to render a background or border color, so I updated the `inset` class in the CSS - I didn't want to spend too much time troubleshooting the cause - maybe we need to update the component library to the latest version?

Reply:
ooohhhh

Reply:
@Mottie I believe you're right. The `va-featured-content` component was renamed to `va-summary-box` in the [latest release](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v37.0.0) which is not yet in vets-website because of the CI issues that were happening. We have the update pending [in this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27945) though.

Reply:
Thanks! @jamigibbs